uploading/models/letterboxd_tables/reviews.js

```diff
@@ -1,10 +1,32 @@
 const db = require("../../../database/dbConfig.js");
 
-module.exports = { addReview };
+module.exports = { 
+  addReview,
+  getReviewById
+};
 
 async function addReview(review) {
-  const [id] = await db("user_letterboxd_reviews").insert(review, "id");
+  await db("user_letterboxd_reviews")
+  .select('*')
+  .where("letterboxd_uri", review.letterboxd_uri)
+  .andWhere("user_id", review.user_id)
+  .then(reviews => {
+    if(reviews.length === 0) {
+      console.log("ADDING REVIEW:", review.review)
```

Comment:
Should we keep these types of console.logs in here for the server? I feel like that could be a pretty big vulnerability if someone malicious got the logs of our server.